_layouts/default.html

```diff
@@ -85,78 +83,16 @@ <h2><span>{{ page.title }}</span></h2>
-                                                {{ category['name'] }} ({{ category['count'] }})
-                                                </a>
-                                            </li>
-                                            {% endfor %}
-                                        </ul> -->
-                                    </aside>
-
-                                    <!-- Recent entries widget-->
-                                    <aside class="widget widget_recent_entries_custom">
-                                        <div class="widget-title">Popular Posts</div>
-                                        <ul>
-                                            <li class="clearfix">
-                                                <div class="wi">
-                                                    <a href="#"><img src="../assets/images/works/img1.jpg" alt="" class="img-fluid"></a>
-                                                </div>
-                                                <div class="wb"><a href="#">Beautiful Day With Friends..</a> <span class="post-date">Jun 15, 2017</span></div>
-                                            </li>
-                                            <li class="clearfix">
-                                                <div class="wi">
-                                                    <a href="#"><img src="../assets/images/works/img2.jpg" alt="" class="img-fluid"></a>
-                                                </div>
-                                                <div class="wb"><a href="#">Nature valley with cooling..</a> <span class="post-date">Jun 10, 2017</span></div>
-                                            </li>
-                                            <li class="clearfix">
-                                                <div class="wi">
-                                                    <a href="#"><img src="../assets/images/works/img3.jpg" alt="" class="img-fluid"></a>
-                                                </div>
-                                                <div class="wb"><a href="#">15 Best Healthy and Easy Salad..</a> <span class="post-date">Jun 8, 2017</span></div>
-                                            </li>
-                                        </ul>
-                                    </aside>
-
-                                    <!-- Text widget-->
-                                    <aside class="widget">
-                                        <div class="widget-title">Text Widget</div>
-
-                                        <p class="text-muted text-widget-des">Exercitation photo booth stumptown tote bag Banksy, elit small batch freegan sed. Craft beer elit seitan exercitation, photo booth et 8-bit kale chips proident chillwave deep v laborum. Aliquip veniam delectus, Marfa eiusmod Pinterest in do umami readymade swag. </p>
-
+                                            <ul>
```

Comment:
Videos and Documentation show up in the widget but their links don't work, is that a known issue for now?

Reply:
Hmm... the categories widget on the right sidebar with video and documentation and their counts work on my end...?